윤원석/feat 8

wonseokyoon opened the pull request:
- 코틀린으로 전환했습니다.
- 기존에 기능 강화한것과 리팩토링한것까지 병합했습니다.
- 사진과 지도가 안찍히는 문제 있습니다

wonseokyoon committed:
test: 삭제 테스트

# Conflicts:
#	.idea/modules.xml
#	apiV1.json
#	application.pid
#	backend/src/test/java/com/NBE_4_5_2/Team5/domain/chat/controller/ChatControllerTest.java
#	backend/src/test/java/com/NBE_4_5_2/Team5/domain/chat/controller/ChatRoomControllerTest.java
#	db_dev.mv.db

# Conflicts:
#	.idea/gradle.xml

wonseokyoon committed:
test: 삭제 테스트

# Conflicts:
#	.idea/modules.xml
#	apiV1.json
#	application.pid
#	backend/src/test/java/com/NBE_4_5_2/Team5/domain/chat/controller/ChatControllerTest.java
#	backend/src/test/java/com/NBE_4_5_2/Team5/domain/chat/controller/ChatRoomControllerTest.java
#	db_dev.mv.db

wonseokyoon committed:
refactor: entity,dto 코틀린 전환

wonseokyoon committed:
chatConroller 코틀린 전환

wonseokyoon committed:
chatRoomConroller 코틀린 전환

wonseokyoon committed:
chatService 코틀린 전환

wonseokyoon committed:
chatRoomService 코틀린 전환

wonseokyoon committed:
Repository 코틀린 전환

wonseokyoon committed:
chatControllerTest 코틀린 전환

wonseokyoon committed:
chatRoomControllerTest 코틀린 전환

omegafrog commented on backend/src/main/java/com/NBE_4_5_2/Team5/domain/chat/controller/ChatController.kt:
```diff
@@ -0,0 +1,84 @@
+                )
+            }
+            .toList()
+        println("===DTO===")
+        println(dtos[dtos.size - 1].lastMessage)
+        println(dtos[dtos.size - 1].lastTimestamp)
+        println(dtos[dtos.size - 1].messageType)
```
`val log = LoggerFactory.getLogger()`로 로깅 라이브러리로 출력하거나 필요없으면 삭제하는게 좋을 것 같습니다.

wonseokyoon replied:
일단 지웠는데, 남아있는 것들은 다음 pr에 반영해서 올리겠습니다.

wonseokyoon committed:
Update ChatController.kt

asungkim committed:
Product 관련 status 추가 및 버그 수정 (#24)

* [feat] 상품 status 추가

* [feat] 페이지에 status 추가

* [refactor] 여러 버그들 수정

* [refactor] 찜, 구매 내역 PreviewResponse 하도록 수정

* schema.d.ts 추적 무시
# Conflicts:
#	.idea/gradle.xml

okjunghyeon committed:
옥정현/fix 26 최근 본 상품 목록 조회, 상품명 검색 기능 수정 + 기타 오류 수정 (#31)

* fix: 회원가입, 회원정보 수정 시 redis에 저장되어 있던 email 인증 정보가 제거되지 않고 만료시간까지 유지되던 버그 수정

- 이메일 인증 완료 후 redis에 해당 데이터를 삭제하도록 로직 수정

* fix: 상품 상세 조회 시 Comment의 필드에 접근하지 못하는 문제 해결

- 기존 Primary constructor를 통해 _target 형태로 선언하고 getter로 반환하는 필드를 Jpa에서 인식하지 못하는 문제 발생
-  Primary 가 아닌 필드로 선언하고 부생성자로 초기화하도록 변경하여 에러 제거
- 상세 페이지 접근 시 ProductPost의 buyer가 lateinit으로 선언되었으나 초기화 되지 않은 상태로 접근하는 오류 발생
  - null 형태로 초기화 되도록 변경

* feat: 최근 본 상품이 제대로 출력되도록 수정

* feat: 최근 본 상품 조회 시 순서대로 정렬되어 출력될 수 있도록 구현

* feat: 메인페이지에서 상품명으로 검색 가능하도록 수정

wonseokyoon committed:
충돌 해결
